Apromore-Extras/APMLogModule/src/test/java/org/apromore/apmlog/AttributeArcDurationTest.java

```diff
@@ -252,4 +255,39 @@ public static void testRetain3(APMLog apmLog, APMLogUnitTest parent) throws Unsu
             parent.printString("'Attribute arc duration (3)' test PASS.\n");
         }
     }
+
+    public static void testAvgDur1(APMLog apmLog, APMLogUnitTest parent) throws UnsupportedEncodingException {
+        APMLogFilter apmLogFilter = new APMLogFilter(apmLog);
+        PLog pLog = apmLogFilter.getPLog();
+        pLog.updateStats();
+        DurSubGraph subGraph = pLog.getAttributeGraph().getNextValueDurations("concept:name", "A");
+        DoubleArrayList durList = subGraph.getValDurListMap().get("A");
+        double avg = getAverage(durList);
+        double twoHours = 1000 * 60 * 60 * 2d;
+        if (avg != twoHours) {
+            throw new AssertionError("TEST FAILED. RESULT MISMATCH.\n expected value=" + twoHours +
+                    "; output value=" + avg);
+        } else {
+            parent.printString("'Attribute arc duration (4) - Average duration' test PASS.\n");
+        }
```

Comment:
JUnit has various assertion methods which could simplify test code like this, something like:

```
import static org.junit.Assert.assertEquals;
```
```
assertEquals(1000 * 60 * 60 * 2d, getAverage(durList));
```

Reply:
I'm not sure where you want to place that code. Please edit it directly.

Reply:
Done!